tests/integration/test_users.py

```diff
@@ -268,23 +269,22 @@ def test_invite_user(self, valid_header):
         assert response.status_code == 200
 
 @pytest.mark.usefixtures('client_class', 'empty_test_db')
-class TestReigsterUser:
+class TestRegisterUser:
     def setup(self):
         self.endpoint = "/api/register"
-        self.valid_payload = {
-            'email': 'faker.unique.email@example.com',
-            'password': 'faker.password'
-        }
-    def test_user_can_register_with_valid_payload(self):
-        response = self.client.post(self.endpoint, json=self.valid_payload)
+        self.valid_payload = user_register_valid_payload
+
+    def test_user_can_register_with_valid_payload(self, user_attributes):
+        response = self.client.post(self.endpoint, json=self.valid_payload(user_attributes()))
 
         assert response.status_code == 201
 
     @patch.object(UserModel, 'create')
-    def test_user_calls_create(self, mock_create):
-        response = self.client.post(self.endpoint, json=self.valid_payload)
+    def test_user_calls_create(self, mock_create, user_attributes):
+        payload = self.valid_payload(user_attributes())
+        response = self.client.post(self.endpoint, json=payload)
 
         mock_create.assert_called_with(
             schema=UserRegisterSchema,
-            payload=self.valid_payload
-        )
\ No newline at end of file
+            payload=payload
+        )
```

Comment:
There should be a blank line at the end of the file.  There wasn't one before, but you might as well add one in before merging this PR.

Reply:
Totally agree

Reply:
hmmm this has a newline at the end of the file. I checked the other files just in case and they all do too.

Reply:
It's not showing up on my machine or on GitHub.
https://github.com/codeforpdx/dwellinglybackend/blob/ns-update_user_schema_tests/tests/integration/test_users.py

Maybe it has it in your local code?

Reply:
I don't see GitHub flagging for a missing newline character. Feel free to push up a change.

Reply:
@NickSchimek just realized I was looking at this file on the _development branch_ locally 🤦‍♂️ 
Your branch has the newline. Sorry about that! 😬

Reply:
😆  no worries. It happens to the best of us.